### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by glaucimar**
Adding "clone your own repo" to the contribution section and making a reference to this section on the "Development setup"  to make it  more clear the steps to be followed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +42,7 @@ pyenv install 3.9.x
 
 Clone the repository
 ```
-git clone git@github.com:HewlettPackard/py-spiffe.git
+Follow though steps in Contributing session.
```
Comment (maxlambrecht) [inline README.md:45]: `Follow through the steps in [Contributing](#Contributing) section`

File: README.md [hunk 2]
```diff
@@ -35,7 +42,7 @@ pyenv install 3.9.x
 
 Clone the repository
 ```
-git clone git@github.com:HewlettPackard/py-spiffe.git
+Follow through the steps in [Contributing](#Contributing) session.
```
Comment (mchurichi) [inline README.md:45]: I think this should be `section` instead of `session`

Comment (glaucimar) [reply README.md:45]: Done, thanks :)

File: README.md (hunk 1 above)
Comment (glaucimar) [reply README.md]: Done, thanks Max!

**Review by maxlambrecht — APPROVED** [review]
💯

**Review by mchurichi — DISMISSED** [review]
LGTM

File: README.md [hunk 3]
```diff
@@ -35,7 +42,7 @@ pyenv install 3.9.x
 
 Clone the repository
 ```
-git clone git@github.com:HewlettPackard/py-spiffe.git
+Follow through the steps in [Contributing](#Contributing) section.
```
Comment (Andres-GC) [inline README.md:45]: One detail, the text is inside a code block so the link style doent's work. It looks like this:

image

On a different note, referring to the whole contributing section looks to me like "exaggerated". there are other steps there that are not required for the purpose of just _get the repository cloned_ as the header suggest.

What about something like (just text, no code block style):

Clone the repository
Follow steps 1 and 2 from the [Contributing](#Contributing) section.

**Review by Andres-GC — COMMENTED** [review]
Thanks Glaucimar. I've added a minor comment for a cosmetic change. As part of that change, would you mind adding Scott to the list of contributors?
`Scott Emmons (HPE)`

File: README.md (hunk 3 above)
Comment (scotte) [reply README.md:45]: Personally, I don't see the need to spell out the steps too explicitly - it's no different than any other github repository.

Comment (Andres-GC) [reply README.md:45]: While I do agree with Scott, I need to mention that the PR was originally created to add step two:
`2- Clone your forked repository (git clone git@github.com:<your_github_account>/py-spiffe.git)`

That is also a common path when working with Github. So if we add that for less experienced persons I think adding the correct steps here will make thing less confusing for them. That was my point here.

Said that, I'm Ok with the text as it is too.

File: README.md [hunk 4]
```diff
@@ -8,11 +8,21 @@ Initial work in progress to create a python library for SPIFFE.
 The _RFC - Py-Spiffe features and use cases_ document is public at: https://docs.google.com/document/d/1IByE9Ge1MTyWD7rL9o99-1c5Fn83p1ra3eApARY9X-I/edit#heading=h.70d940e8vk81
 
 ## Initial contributors
```
Comment (mchurichi) [inline README.md:10]: super nit picky, but wondering if we should standarize on "FirstName LastName", my OCD isn't letting me see this 😂 
@Andres-GC @maxlambrecht would you mind?

Comment (Andres-GC) [reply README.md:10]: +1 to list all names in the same format. 
I don't have any preference between Name-Lastname or Lastname-Name

File: README.md (hunk 3 above)
Comment (glaucimar) [reply README.md]: Done!

File: README.md (hunk 4 above)
Comment (glaucimar) [reply README.md:10]: Thanks folks! Done!

**Review by Andres-GC — APPROVED** [review]
:ice_cream:

**Review by mchurichi — APPROVED** [review]

**Pull request merged**